@Types/ITechRecord.d.ts

```diff
@@ -82,54 +85,60 @@ interface Axle {
     };
 }
 
-export interface AdrDetails {
-    vehicleDetails: VehicleDetails;
-    permittedDangerousGoods: PERMITTED_DANGEROUS_GOODS[];
-    additionalExaminerNotes: string;
-    applicantDetails: ApplicantDetails;
-    memosApply: MEMOS_APPLY[];
-    additionalNotes: AdditionalNotes;
-    tank: Tank;
-}
-
-export interface AdditionalNotes {
-    number: number;
-    guidanceNotes: string[];
-}
-
-export interface ApplicantDetails {
-    name: string;
-    street: string;
-    town: string;
-    city: string;
-    postcode: string;
-}
-
-export interface Tank {
-    tankDetails: TankDetails;
-    tankStatement: TankStatement;
-}
-
-export interface TankDetails {
-    tankManufacturer: string;
-    tc2IntermediateApprovalNo: string;
-    tc2IntermediateExpiryDate: Date;
-    tc3PeriodicNumber: string;
-    tc3PeriodicExpiryDate: Date;
-    yearOfManufacture: string;
-    tankCode: string;
-    specialProvisions: string;
-    tankManufacturerSerialNo: string;
-    tankTypeAppNo: string;
-}
-
-export interface TankStatement {
-    substancesPermitted: SUBSTANCES_PERMITTED;
-    statement: string | null; // statement and productList are mutually exclusive
-    productList: string | null;
+interface AdrDetails {
+    vehicleDetails: {
+        type: string,
+        approvalDate: string
+    },
+    listStatementApplicable?: boolean,
+    batteryListNumber?: string,
+    declarationsSeen?: boolean,
+    brakeDeclarationsSeen?: boolean,
+    brakeDeclarationIssuer?: string,
+    brakeEndurance?: boolean,
+    weight?: string,
+    compatibilityGroupJ?: boolean,
+    documents?: string[],
+    permittedDangerousGoods: string[],
+    additionalExaminerNotes?: string,
+    applicantDetails: {
+        name: string,
+        street: string,
+        town: string,
+        city: string,
+        postcode: string
+    },
+    memosApply?: string[],
+    additionalNotes?:{
+        number?: string[],
```

Comment:
this change will affect the mobile app. initially, it was "number": number (it can also be string). the issue is that now this property is an array. why does a property called "number" (not "numbers") is an array and not a single value?

Reply:
This is how the latest excel with the AdrFields looks like. I just followed that, wrote the API specs and implemented them. I think these issues should be discussed with someone else, I'm not in position to answer to this. If you want to take a look at the excel I'm talking about, you'll find it attached in ticket 8714 along with the latest version of the api spec.

Reply:
yes, i totally agree. i am just leaving this comment here because @arupdvsa said mobile app won't be affected by the changes. it actually is.

Reply:
I suggested impact will be minimal and insignificant